tests/test_validation.py

```diff
@@ -14,6 +14,12 @@
                                                     '..', 'example_listings'))
 
 
+def drop_api_exported_fields_so_that_api_import_will_validate(data):
```

Comment:
The example listings contain the full data for service requests, not just fields that get stored to `.data`. I believe changing them would unfortunately break a lot of tests that don't mix all of the request fields into request data, so keeping the helper method here was an easier solution for now.